cfgov/v1/models/molecules.py

```diff
@@ -64,6 +64,47 @@ class Meta:
         template = 'v1/demo/molecules/image_text_5050.html'
 
 
+class ImageText2575(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(blank=True)
```

Comment:
Yikes. That should be `required=False`. This was copied from the other ImageText class which is wrong. I'll change this one in this PR and the other one in a clean up PR at a later date. I don't know if this should be required or not.